packages/ember-data/tests/integration/relationships/belongs_to_test.js

```diff
@@ -7,6 +7,10 @@ var hasMany = DS.hasMany;
 var belongsTo = DS.belongsTo;
 var hash = Ember.RSVP.hash;
 
+function getComputedPropertyDesc(model, key) {
+  return Ember.meta(model).descs ? Ember.meta(model).descs[key] : model[key];
```

Comment:
the check is a bit more specific.

cc @ebryn I know this is violating a private API, but it is working around some annoying cross version issues and a serious performance hack. The proper solution is a hefty refactor that will happen sometime soon. Hence the triage fix.